### DISCUSSION TITLE
圖靈測試

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS3073-C/Python-Community-GS3073-C#742] [General]
對不起可能是我孤陋寡聞嗚嗚嗚
今天才真正去查圖靈測試是什麼
查完之後想的第一件事情是
說不定某些主播或小網紅可以買高度擬人的聊天機器人
（或其實現在真的有人會這麼做
這樣不僅可以把時間分給其他事還同時可以賺錢
（挖勒會不會太壞哈哈哈
[https://wiki.mbalib.com/zh-tw/图灵测试](https://wiki.mbalib.com/zh-tw/%E5%9B%BE%E7%81%B5%E6%B5%8B%E8%AF%95)

**Comment by jerry0108566** [comment]
其實驗證碼當初也是圖零測試的一種反向應用，他就是利用機器人無法分辨的一些扭曲圖片來確認你是否是人類。
